src/gc/env/gcenv.ee.h

```diff
@@ -82,6 +82,10 @@ class GCToEEInterface
     static bool CreateThread(void (*threadStart)(void*), void* arg, bool is_suspendable, const char* name);
     static void WalkAsyncPinnedForPromotion(Object* object, ScanContext* sc, promote_func* callback);
     static void WalkAsyncPinned(Object* object, void* context, void(*callback)(Object*, Object*, void*));
+    static void FireGcStartAndGenerationRanges(uint32_t count, uint32_t depth, uint32_t reason, uint32_t type);
```

Comment:
Nit: It would be nice to separate blocks of the related methods with new-lines.

Reply:
will do!